apps/mainsite/views.py

```diff
@@ -88,6 +88,9 @@ def upload(req):
 @authentication_classes([TokenAuthentication, SessionAuthentication, BasicAuthentication])
 @permission_classes([IsAuthenticated])
 def aiskills(req, searchterm):
+    # The searchterm is encoded URL safe, meaning that + and / got replaced by - and _
+    searchterm.replace("-", "+")
```

Comment:
I believe this should be ```searchterm = searchterm.replace``` because replace doesnt modify the string in place

Reply:
Fixed